03-transactions.md

```diff
@@ -117,17 +117,76 @@ If a revoked commitment transaction is published, the other party can spend this
 
 #### `to_remote` Output
 
-This output sends funds to the other peer and thus is a simple P2WPKH to `remotepubkey`.
+This output sends funds to the other peer, thus is not encumbered by a
+revocation private key.
+
+If `option_anchor_outputs` applies to the commitment transaction, the `to_remote` output is delayed similarly to the `to_local` output, and is to a fixed key:
+
+    `max_to_self_delay`
```

Comment:
If we're changing this script entirely, I believe @ariard explained to use that you don't need to put that CSV constrain in the script when it's not in a conditional branch, it can simply be on the `nSequence` field itself (and the script remains just a P2WPKH). 
Can you confirm @ariard ?